app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreen.kt

```diff
@@ -2,15 +2,265 @@ package com.kks.nimblesurveyjetpackcompose.ui.presentation.home
+                top.linkTo(date.bottom, 4.dp)
+                start.linkTo(parent.start, 20.dp)
+            },
+            fontSize = 34.sp,
+            fontWeight = FontWeight.Bold
+        )
+        UserIcon(
+            modifier = Modifier
+                .clip(CircleShape)
+                .size(36.dp)
+                .constrainAs(userImage) {
+                    end.linkTo(parent.end, 20.dp)
+                    top.linkTo(parent.top, 79.dp)
+                }
+        )
+        BottomView(
+            modifier = Modifier
+                .fillMaxWidth()
+                .constrainAs(bottomView) { bottom.linkTo(parent.bottom, 54.dp) },
+            numberOfPage = numberOfPage,
+            currentPage = selectedSurveyNumber,
+            surveyResponse = surveyResponse
+        )
+    }
+}
+
+@Composable
+fun UserIcon(modifier: Modifier) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_launcher_foreground),
+        contentDescription = stringResource(id = R.string.home_user_image),
+        modifier = modifier
+    )
+}
+
+@Composable
+fun BottomView(
+    modifier: Modifier,
+    surveyResponse: SurveyResponse,
+    numberOfPage: Int,
+    currentPage: Int
+) {
+    Column(modifier = modifier) {
+        DotsIndicator(
+            totalDots = numberOfPage,
+            selectedIndex = currentPage,
+            selectedColor = Color.White,
+            unSelectedColor = White20,
+            indicatorSize = 8.dp,
+            space = 5.dp
+        )
+        Spacer(modifier = Modifier.height(26.dp))
+        SurveyText(
+            text = surveyResponse.attributes?.title.orEmpty(),
+            fontSize = 28.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier.padding(start = 20.dp, end = 20.dp)
+        )
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            SurveyText(
+                text = surveyResponse.attributes?.description.orEmpty(),
+                fontSize = 17.sp,
+                color = White70,
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(start = 20.dp, end = 20.dp)
+            )
+            Button(
+                onClick = { },
+                modifier = Modifier
+                    .clip(CircleShape)
+                    .size(56.dp),
+                colors = ButtonDefaults.buttonColors(backgroundColor = Color.White)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_baseline_arrow_forward_ios_24),
+                    contentDescription = stringResource(id = R.string.home_survey_detail_button)
+                )
+            }
+            Spacer(modifier = Modifier.width(20.dp))
+        }
+    }
+}
+
+@Composable
+fun SurveyText(
+    text: String,
+    modifier: Modifier = Modifier,
+    fontSize: TextUnit,
```

Comment:
`Modifier` should be the first optional parameter
```suggestion
    fontSize: TextUnit,
    modifier: Modifier = Modifier,
```